requirements.txt

```diff
@@ -6,7 +6,7 @@ matplotlib==3.5.1
 opt-einsum==3.3.0
 line-profiler==4.0.3
 einops==0.4.0
-tensorflow==2.8.0
+tensorflow>=2.8.0
```

Comment:
this is tough, what's the reason for dropping the exact version?

Reply:
Fixed.